devel/wolfssl/Portfile

```diff
@@ -6,7 +6,7 @@ name                wolfssl
 version             5.0.0
 categories          devel security
 platforms           darwin
-maintainers         yassl.com:chris
+maintainers         {wolfssl.com:facts @JacobBarthelmeh}
```

Comment:
can `openmaintainer` also be added so that minor changes to the Portfile can be made without your explicit permission?